pkg/api/kubernetesagentpool/v20170727/types.go

```diff
@@ -0,0 +1,114 @@
+package v20170727
```

Comment:
See my note here https://github.com/Azure/acs-engine/pull/957#discussion_r130453111 explaining the subtle differences in the structs, and the fact that this code is being called at a unique point during an HTTPs transaction with the ACS RP